packages/manager/modules/pci/src/projects/project/instances/stop/stop.routing.js

```diff
@@ -0,0 +1,22 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.instances.stop', {
+    url: '/stop?instanceId',
+    views: {
+      modal: {
+        component: 'pciInstancesInstanceStop',
```

Comment:
Relates to following module `@ovh-ux/ng-ui-router-layout`.
Isn't supposed to be defined as a dependency of following module: `ovhManagerPciInstancesStop`?

Reply:
Deps. has been added